### DISCUSSION TITLE
B站限定舰长回

### DISCUSSION THREAD

**Discussion by wardhardack** [HMBSbige/BilibiliLiveRecordDownLoader#286] [General]
3月14日情人节，我关注的一个主播弄了舰长限定直播，一开始是可以免费进去看得到舰长付费直播内容，但是切断直播，或者是被超管切断，再次连接就进不去了，录制的是路人画面，没有舰长付费画面。

**Comment by wardhardack** [comment]
我用的是3.11.0版本，不知道 现在3.12.4是不是 已经没有这个问题了

**Comment by HMBSbige** [comment]
不是很懂，如果是付费直播登录就行了

**Reply by wardhardack** [reply]
付费直播登录就可以录制了吗，即使我账号没有付费

**Reply by HMBSbige** [reply]
怎么可能啊，这是录制直播，直播怎么样录制就怎么样
